ckanext/depositar_theme/fanstatic/webassets.yml

```
@@ -29,3 +35,20 @@ field-extra-js:
 p5-js:
   output: depositar_theme/%(version)s_p5.js
   contents: vendor/p5.js/p5.js
+
+font-awesome-css:
+  output: depositar_theme/%(version)s_font-awesome.css
+  filters: cssrewrite
+  contents: font-awesome/css/font-awesome.css
+
+jquery-js:
+  ouput: depositar_theme/%(version)s_jquery.js
+  contents: vendor/bootstrap-4.js/jquery.min.js
+
+bootstrap-4-js:
```

加上 `rjsmin` 於 production 環境產生 minified js

```suggestion
bootstrap-4-js:
  filters: rjsmin
```